### ISSUE TITLE
deposit successful is displayed to early

### ISSUE THREAD

**Issue by maxjuno** [Wasmswap/wasmswap-interface#164]
At the moment we are just checking if there was a transaction and the ui shows deposit successful.
But the deposit could still fail because the relayer is failing or for whatever other reason (example: https://www.mintscan.io/cosmos/txs/7332666DC0F51015ED82F79ED2182D7CFCE330251CBE29A9302D981E4D055085 after this tx it shows also successful)

Maybe we should implement a separate section where you can see pending transfers

**Comment by sashimi36** [comment]
Yes thanks for pointing this out. :) This is something we have on the roadmap. I'm not sure tho if we have a certain timeline on that just yet
